exonum-java-binding/core/rust/src/runtime/java_service_runtime.rs

```diff
@@ -118,24 +127,36 @@ impl JavaServiceRuntime {
 
     /// Adds required EJB-related arguments to JVM configuration
     fn add_required_arguments(
```

Comment:
I am not quite happy with this method, it got too complex and requires refactoring.

Reply:
🤷‍♂️ I tried to avoid rewriting the whole stuff during merge